### PR TITLE
feat(monitoring): Filter by file type

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by ioanpocol** [comment]
Some CSS issues will be fixed by Aco

File: client/app/scripts/superdesk-desks/aggregate.js [hunk 1]
```diff
@@ -66,6 +68,44 @@
...
+        this.hasFileType = function(fileType) {
+            return this.selectedFileType === fileType;
+        };
+
+        /**
+         * Set the current 'fileType' filter
+         * param {string} fileType
+         */
+        this.setFileType = function(fileType) {
+            console.log('set filter: ', fileType);
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-desks/aggregate.js]: This can be removed.

File: client/app/scripts/superdesk-desks/aggregate.js [hunk 2]
```diff
@@ -26,6 +26,8 @@
         this.searchLookup = {};
         this.deskLookup = {};
         this.stageLookup = {};
+        this.fileTypes = ['all', 'text', 'photo', 'composite', 'video', 'audio'];
```
Comment (amagdas) [inline client/app/scripts/superdesk-desks/aggregate.js]: Do we need preformatted as well?

**Comment by ioanpocol** [comment]
Closed because opened PR 905 that contains CSS/HTML improvements.

**Pull request closed without merging**